utils/appliance.py

```diff
@@ -636,6 +647,51 @@ def precompile_assets(self, log_callback=None):
 
         return status
 
+    def backup_database(self, log_callback=None):
+        """Backup VMDB database
+
+        TO DO:
+            Remove hard coded NFS server name and get NFS server details from yaml file.
+
+        """
+        if log_callback is None:
+            log_callback = logger.info
+
+        log_callback('Backing up database')
+
+        client = self.ssh_client()
+        status, output = client.run_rake_command(
+            'evm:db:backup:remote -- --uri nfs://10.16.7.91/home/jenkins --remote-file-name backup')
```

Comment:
Hmm, I don't think exposing internal IP addresses is a good idea. You should rather create (even a temporary) entry in the yaml. And I believe this NFS server is in YAML already.